.github/actions/check_docstrings/action.yml

```diff
@@ -0,0 +1,56 @@
+name: Check Docstrings
+inputs:
+    python-version:
+        description: 'The version of Python to use for the workflow.'
+        default: '3.10'
+        required: false
+        type: string
+    repository:
+        description: 'The repository to check the docstrings for.'
```

Comment:
An example in the descriptive would be useful. It is not clear to me that I should expect the /owner/repo notation from here.